src/booleans.js

```diff
@@ -1,62 +1,98 @@
-function negate(a) {
-  // your code here
-};
+const negate = (a) => !a;
 
-function both(a, b) {
-  // your code here
-};
+const both = (a, b) => {
+  if (!a || !b) {
```

Comment:
You could use `return Boolean(a || b)` or `return !!(a || b)` here if you wanted to get rid of the if else statement.